BestBuyCRUD/EmployeesRepo.cs

```diff
@@ -0,0 +1,168 @@
+    {
+        public string connectionString; // field
+
+        public EmployeesRepo(string conn)
+        {
+            connectionString = conn;
+        }
+
+        public List<Employees> GetEmployees()
+        {
+            MySqlConnection connect = new MySqlConnection(connectionString);
+
+            using (connect)
+            {
+                connect.Open();
+
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "Select EmployeeId AS ID, FirstName AS First, MiddleInitial AS Middle, LastName AS Last," +
+                    "EmailAddress AS Email, PhoneNumber As Number, Title, DateOfBirth AS Bday FROM Employees";
+
+                MySqlDataReader reader = cmd.ExecuteReader();
+
+                List<Employees> employees = new List<Employees>();
+                while(reader.Read())
+                {
+                    Employees emp = new Employees
+                    {
+                        EmployeeId = (int)reader["ID"],
+                        FirstName = reader["First"].ToString(),
+                        MiddleInitial = reader["Middle"].ToString(),
+                        LastName = reader["Last"].ToString(),
+                        EmailAddress = reader["Email"].ToString(),
+                        PhoneNumber = reader["Number"].ToString(),
+                        Title = reader["Title"].ToString(),
+                        DateOfBirth = Convert.ToDateTime(reader["Bday"])
+                    };
+                    employees.Add(emp);
+                }
+                return employees;
+                
+            }
+         }
+
+        public void CreateEmployees(Employees createEmployees)
+        {
+            MySqlConnection connect= new MySqlConnection(connectionString);
+
+            using (connect)
+            {
+                connect.Open();
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "INSERT INTO employees (FirstName, MiddleInitial," +
+                    "LastName, EmailAddress, PhoneNumber, Title, DateOfBirth) VALUES" +
+                    "( @First, @Middle, @Last, @Email, @Number,@Title,@Bday);";
+               // cmd.Parameters.AddWithValue("ID", createEmployees.EmployeeId);
+                cmd.Parameters.AddWithValue("First", createEmployees.FirstName);
+                cmd.Parameters.AddWithValue("Middle", createEmployees.MiddleInitial);
+                cmd.Parameters.AddWithValue("Last", createEmployees.LastName);
+                cmd.Parameters.AddWithValue("Email", createEmployees.EmailAddress);
+                cmd.Parameters.AddWithValue("Title", createEmployees.Title);
+                cmd.Parameters.AddWithValue("Number", createEmployees.PhoneNumber);
+                cmd.Parameters.AddWithValue("BDay", createEmployees.DateOfBirth);
+
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+
+        public void UpdateEmployees(int eId,string fn, string ln, string m )
```

Comment:
"eId", "fn", "ln", "m" are not very descriptive names for what this method is supposed to be supplied when called